components/tag/selectableList/src/index.js

```diff
@@ -0,0 +1,109 @@
+import React, {Component, PropTypes} from 'react'
+import TagSelectable from '@schibstedspain/sui-tag-selectable'
+
+const removeFromArray = (arr, value) => {
+  return arr.filter(item => value !== item)
+}
+
+export default class TagSelectableList extends Component {
+  state = {
+    selectedValues: []
+  }
+
+  constructor (...props) {
+    super(...props)
+    this.tagHandlers = this.props.tagsList.map((tag) => this.toggleValue.bind(this, tag.value))
+    this.allHandler = this.toggleAll.bind(this)
+  }
+
+  toggleValue (value) {
+    let selectedValues = this.state.selectedValues
+    if (selectedValues.indexOf(value) !== -1) {
```

Comment:
https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Array/includes
Be aware of compatibility.